packages/mwp-app-render/src/util/seo/__snapshots__/ldJson.test.js.snap

```diff
@@ -7,6 +7,7 @@ Object {
   "description": "The coolest event in the world during which we will run and dance and sing
 	<script>alert(\\"bad time\\")</script>, 😊, &lt;blink&gt;what what&lt;blink&gt; this is getting
 	longer than it needs to be why am I still typing omg",
+  "endDate": "1970-01-18T10:01+00:00",
```

Comment:
is this the right time? I think the event object mock used seconds rather than milliseconds, but we should find out what events _usually_ provide and make sure the test matches up with typical usage

Reply:
Yeah, my understanding here is `MOCK_EVENT` has a `0` `utc_offset` but we can mock this with a valid `utc_offset` in milliseconds.